src/main.tsx

```diff
@@ -24,11 +25,11 @@ const router = createBrowserRouter([
   {
     path: "/thankyou",
     element: (
-      <div className=" w-full h-svh flex items-center justify-center text-center p-4">
-        <h1 className=" text-3xl">
-          Makasih yaa udah main <br />
-          🩷🩷🩷
-        </h1>
+      <div className=" w-full h-screen flex flex-col items-center justify-center">
+        <h1 className=" text-3xl">Makasih yaa udah main 🩷🩷🩷</h1>
+        <Link to="/" className="mt-4">
```

Comment:
for now we don't need to have a play again button because the content is just static

Reply:
Got it